README.md

```diff
@@ -315,14 +315,20 @@ $connector->connect('tls://smtp.googlemail.com:465')->then(function (ConnectionI
 
 ### HTTP requests
 
-HTTP operates on a higher layer than this low-level SSH proxy implementation.
-If you want to issue HTTP requests, you can add a dependency for
-[clue/reactphp-buzz](https://github.com/clue/reactphp-buzz).
-It can interact with this library by issuing all
-[HTTP requests through an SSH proxy server](https://github.com/clue/reactphp-buzz#ssh-proxy).
-When using the `SshSocksConnector` (recommended), this works for both plain HTTP
-and TLS-encrypted HTTPS requests. When using the `SshProcessConnector`, this only
-works for plaintext HTTP requests.
+This library also provides you with the usage of [HTTP requests through an SSH proxy server](https://github.com/reactphp/http#ssh-proxy).
+
+To send out HTTP requests you have to add a dependency for [ReactPHP's async HTTP client](https://github.com/reactphp/http). You can then use this HTTP client in combination with the `SshSocksConnector`. This works for both plain HTTP and TLS-encrypted HTTPS requests.
```

Comment:
```suggestion
This library also allows you to send HTTP requests through an SSH proxy server.

In order to send HTTP requests, you first have to add a dependency for [ReactPHP's async HTTP client](https://github.com/reactphp/http#client-usage). You can then use this HTTP client in combination with the [`SshSocksConnector`](#sshsocksconnector) provided by this package. This allows you to send both plain HTTP and TLS-encrypted HTTPS requests like this:
```

Reply:
Your version sounds better, thanks for the advice :+1: